src/pages/lessons/[slug].tsx

```diff
@@ -267,7 +318,21 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
                     <NextUpOverlay
                       lesson={lesson}
                       send={send}
-                      url={next_up_url}
+                      nextUp={nextUp}
+                    />
+                  </OverlayWrapper>
+                )}
+                {playerState.matches('rating') && (
+                  <OverlayWrapper>
+                    <RateCourseOverlay
+                      course={lesson.course}
+                      onRated={() => {
+                        // next in this scenario needs to be considered
+                        // we should also consider adding the ability to
+                        // comment
+                        send('NEXT')
```

Comment:
we had combined the rating and comment previously, and maybe that's what we need to do here as well?

Reply:
dunno what to display when it's all done, probably a `RECOMMEND` overlay?